playbook/app/pb_kits/playbook/pb_date_range_inline/_date_range_inline.jsx

```diff
@@ -49,6 +49,7 @@ const DateRangeInline = (props: DateRangeInlineProps) => {
       <If condition={icon}>
         <Body
             color="light"
+            key={startDate.toString()}
```

Comment:
What if there are two of the same dates?  Could we add a random number to this to ensure that is never the case?

Reply:
Several of the dates passed in the example were similar so didn't think there was a collision issue within the kit itself but I can add a random number just in case 👍